cmd/fleetctl/get.go

```diff
@@ -883,6 +902,20 @@ func printTable(c *cli.Context, columns []string, data [][]string) {
 	table.Render()
 }
 
+func printKeyValueTable(c *cli.Context, rows [][]string) {
+	table := borderlessTabularTable(c.App.Writer)
+	table.AppendBulk(rows)
+	table.Render()
+}
+
+func printColoredMsg(c *cli.Context, msg string, color int) {
```

Comment:
It seems like we are (ab)using the table package to print colors. Perhaps it would be a good idea to use a dedicated package for this with a simpler api. For example github.com/fatih/color.

Reply:
Yeah that's fair, I didn't want to introduce a new dependency just for that when this existing one supports it, but it's true that it is a bit abusing the table package.

Reply:
Ha, looking at `go.mod`, looks like `fatih/color` is already a dependency. Pretty sure I did check for that, no idea how I missed it!

Reply:
Yeah, it's not really used much at the moment, only in `server/fleet/utils.go`